lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -251,7 +251,7 @@ function logChildProcessError(localConsole, err) {
     localConsole.adoptStdStrings(err);
   }
 
-  localConsole.log(log.redify('Error: ') + err + '\n' + err.stack);
+  localConsole.log(log.redify(err.stack || err.message));
```

Comment:
Drive by: `err.stack` already includes `err.message` at the start, so this results in us printing the error message twice. Also, most errors already have "Error:" at the start so I thought it would make more sense to just redify the whole thing.